src/index.js

```diff
@@ -0,0 +1,78 @@
+import './style.css';
+
+const pressEnter = document.getElementById('press-enter');
+const hoverText = document.getElementById('button-hover');
+
+const showToDo = document.getElementById('task-list');
+const myTask = document.getElementById('my-task');
+const form = document.getElementById('form');
+const error = document.getElementById('error');
+
+// Hover message when hover over enter icon
+pressEnter.addEventListener('mouseover', () => {
+  hoverText.style.display = 'block';
+});
+
+pressEnter.addEventListener('mouseout', () => {
+  hoverText.style.display = 'none';
+});
+
+// Local Storage
+class ToDo {
+  constructor() {
+    this.taskList = JSON.parse(localStorage.getItem('storage-task')) || [];
```

Comment:
- [ ] You should create an array of objects with `description`, `index`, and `completed`. in this step of the project you are not required to use local storage, kindly make sure to use a hard-coded array instead
e.g
```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```